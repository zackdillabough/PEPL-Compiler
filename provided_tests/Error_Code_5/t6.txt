POLY A(R) = R + 2 R + 1;
POLY C(Q, W, E, R, T, Y) = Q^121 + W - R^2 - 1;
POLY B(R) = R + 2 R + 1;
START
A(3);
INPUT X;
B(X);

C(X, Y, Z, 
Z, Y, Z);

B(P);

C(X, C(X, Y, Z, X, Y, Z), C(X, Y, Z, X, Y, Z), C(X, Y, Z, X, Y, Z), B(X), 1);


INPUT Y;
INPUT Z;
INPUT P;
INPUT Q;
INPUT R;

C(X, Y, Z, P, Q, R);

9 8 6 1 3 4
