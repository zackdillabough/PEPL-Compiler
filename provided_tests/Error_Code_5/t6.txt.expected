Error Code 5: 9 9 10 10 10 12 14 14 14 14 14 14 14 14 14 14 14 14 